tmt/cli.py

```diff
@@ -353,20 +357,20 @@ def show(context, **kwargs):
         echo()
 
 
-@tests.command()
+@tests.command('lint')
 @click.pass_context
 @name_filter_condition
-@click.option(
-    '-f', '--fix', is_flag=True,
-    help='Attempt to fix all discovered issues.')
+@fix
 @verbose_debug_quiet
-def lint(context, **kwargs):
+def lint_tests(context, **kwargs):
     """
     Check tests against the L1 metadata specification.
 
     Regular expression can be used to filter tests for linting.
     Use '.' to select tests under the current working directory.
     """
+    # Workaround https://github.com/pallets/click/pull/1840 for click 7.x
+    context.params.update(kwargs)
```

Comment:
The tests are now green, ey \o/. The reason why it was passing locally and not in testing farm was a difference between click 7 and click 8 in forwarding params - see https://github.com/pallets/click/pull/1840 . I think my workaround is the least painful solution since it only affects 3 subcommands, but I'd also like to know your opinion @adka1408 @psss

Reply:
Sounds good, let's prepend with a `FIXME` prefix to clean up later when fresh `click` gets to the repositories.

Reply:
Fixed, will merge once the tests finish, thanks